Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGreyExampleSwiftTests.swift

```diff
@@ -195,9 +195,8 @@ class EarlGreyExampleSwiftTests: XCTestCase {
 
   func testWithCondition() {
     let myCondition = GREYCondition.init(name: "Example condition", block: { () -> Bool in
-      var i = 1
-      while i <= 100000 {
-        i++
+      for j in 0...100000 {
```

Comment:
That's not what I meant. The impression I get is that the while loop will iterate from 1 to 100000 (include), and the for loop will iterate from 0 to 100000 (inclusive). Is that right?